content/authors/irene-l-tanner/_index.md

```diff
@@ -1,37 +1,48 @@
 ---
 organizations:
-- name:
-  url: ''
-roles:
-  - Coordinator
-teams:
-  - sp19
-  - sp20
-  - fa20
+  - name: Center for Research in Computer Vision
+    url: ""
 github: irene-l-tanner
 authors:
-- irene-l-tanner
+  - irene-l-tanner
 user_groups:
-- Coordinator
-- sp19-core-coordinator
-- sp19-coordinator
-- sp20-gbms-coordinator
-- sp20-coordinator
-- fa20-coordinator
-- fa20-discussions-coordinator
-- fa20-gbms-coordinator
+  - Coordinator
+  - sp19-core-coordinator
+  - sp19-coordinator
+  - sp20-gbms-coordinator
+  - sp20-coordinator
+  - fa20-coordinator
+  - fa20-discussions-coordinator
+  - fa20-gbms-coordinator
 title: Irene Tanner
-role: Coordinator
+role: Director
 interests:
-- '...'
-bio:
+  - Health Informatics
+  - Data Analysis
+  - Computer Vision
+bio: Hi, I'm Irene. I'm an NSF Graduate Research Fellow conducting research in
+  the Center for Research in Computer Vision on automatically detecting Burmese
+  Pythons in the Everglades. As an undergraduate, I served as Coordinator and
+  Social Director of AI@UCF and conducted research in medical image analysis.
 social:
-- icon: github
-  icon_pack: fab
-  link: https://github.com/irene-l-tanner
+  - icon: github
+    icon_pack: fab
+    link: https://github.com/irene-l-tanner
+  - link: https://www.linkedin.com/in/irene-l-tanner/
+    icon_pack: fab
+    icon: linkedin
 education:
   courses:
-  - course: BS in <UNK>
-    institution: Univeristy of Central Florida
-    year: YYYY
+    - course: BS in Computer Science
+      institution: Univeristy of Central Florida
+      year: 2016
+    - course: PhD in Computer Engineering
+      institution: University of Central Florida
+teams:
+  - sp19
+  - sp20
+  - fa20
+roles:
+  - Coordinator
 ---
+Check out my website at ireneltanner.com :)
```

Comment:
You can add your own website as an icon if you want:
```yaml
- link: <url>
  icon_pack: fas
  icon: globe-americas
```

Also, if you want the emoji, use `:slightly_smiling_face:`.